lib/Runtime/Language/DynamicProfileStorage.cpp

```diff
@@ -296,7 +296,7 @@ char16 const * DynamicProfileStorage::GetMessageType()
 
 bool DynamicProfileStorage::Initialize()
 {
-    AssertMsg(!initialized, "Initialize called multiple times");
```

Comment:
This change was supposed to be an definition check against _WIN32. For xplat this function is safe to call repeatedly.